app/controllers/members_controller.rb

```diff
@@ -0,0 +1,64 @@
+class MembersController < ApplicationController
+	before_action :require_login
+
+	def new
+		@member = Member.new
+	end
+
+	def create
+		@member = Member.new(member_params)
+		@member.family_id = current_user.id
+		@member.save
+		@member.assign_color
+		@member.save
```

Comment:
Why save twice here - is the color dependent on the family in some way? If so, should it be part of a callback on save?